### ISSUE TITLE
Add support for `font-weight` ranges in `@font-face`

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#4327]
> Clearly describe the bug

There are false positives when using `font-weight` inside `@font-face`

https://github.com/stylelint/stylelint/tree/master/lib/rules/font-weight-notation

Reference:

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face#Descriptors
> `font-weight`
> >  A `font-weight` value. Since Firefox 61 (and in other modern browsers) this also accepts two values to specify a range that is supported by a `font-face`, for example `font-weight: 100 400;`

> Which rule, if any, is the bug related to?

`font-weight-notation`

> What CSS is needed to reproduce the bug?

```css

@font-face {
	font-family: "Inter var";
	font-weight: 100 900;
	font-style: normal;
	src: url(./assets/fonts/inter/Inter-upright.var.woff2) format("woff2");
}

```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "font-weight-notation": [ "numeric", {
      ignore: ["relative"],
    } ],
  }
}
```

> Which version of stylelint are you using?

11.0.0

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `./node_modules/.bin/stylelint style.css --config=.stylelintrc.json`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
style.css
260:15  ✖  Expected numeric font-weight notation font-weight-notation
```

**Comment by fanich37** [comment]
I would like to work on this issue if it's still relevant and available.

**Comment by vankop** [comment]
@fanich37 Yes, it is still relevant, PRs always welcome 🤗
